.github/workflows/build-release.yml

```diff
@@ -204,14 +204,11 @@ jobs:
       - name: Release
         # Naively, only create GitHub Release object for purported non-prerelease versions
         if: ${{ !contains(inputs.releaseVersion, '-') }}
-        uses: softprops/action-gh-release@v1
+        uses: ncipollo/release-action@v1
         with:
-          prerelease: true
-          generate_release_notes: true
-          fail_on_unmatched_files: true
-          tag_name: "${{ inputs.releaseVersion }}"
+          makeLatest: true
+          generateReleaseNotes: true
+          artifactErrorsFailBuild: true
+          tag: "${{ inputs.releaseVersion }}"
           name: "v${{ inputs.releaseVersion }}"
-          files: |
-            *.csv
-            *.json
-            *.xlsx
+          artifacts: "*.csv,*.json,*.xlsx"
```

Comment:
Can you point me to a release that you've created with this action? If I could see that, supporting this would be easier than falling in love 😉

Reply:
- [`Build Release` run specifying a valid release version](https://github.com/CMSgov/beneficiary-fhir-data/actions/runs/7559001057)
   -  The corresponding GitHub Release was deleted after this run
- [`Build Release` run specifying a non-release version](https://github.com/CMSgov/beneficiary-fhir-data/actions/runs/7559275115)